internal/exec/exec_public_test.go

```diff
@@ -36,19 +36,16 @@ type ExecManagerPublicTestSuite struct {
 	suite.Suite
 }
 
-func (suite *ExecManagerPublicTestSuite) NewTestExecManager(
-	debug bool,
-) internal.ExecManager {
+func (suite *ExecManagerPublicTestSuite) NewTestExecManager() internal.ExecManager {
```

Comment:
Can we keep the formatting the same here.

Reply:
![fcd](https://github.com/retr0h/go-gilt/assets/1404587/452dad32-82f5-4628-91d0-f5ea68df6f9f)

I can try, but both VS Code and `gofumpt` insist on the code looking like this, so forcing otherwise it will fail unit tests.